File: 1-Iniciante/1013-O-Maior/1013.py
a, b, c = input().split()
a = int(a)
b = int(b)
c = int(c)
if (a + b + abs(a - b)) / 2 > c:
    print (int((a + b + abs(a - b)) / 2), "eh o maior")
else:
    print (c, "eh o maior")

File: 1-Iniciante/1013-O-Maior/1013.c
#include <stdio.h>
#include <stdlib.h>

int main() {

    int a, b, c;
    scanf ("%d %d %d", &a, &b, &c);
    if (((a + b + abs(a - b)) / 2) > c) {
        printf ("%d eh o maior\n", (a + b + abs(a - b)) / 2);
    }
    else {
        printf ("%d eh o maior\n", c);
    }

    return 0;
}